### PR TITLE
Initial boilerplate setup and run

### PR DESCRIPTION
**Pull request by 2uasimojo**

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -6,20 +9,13 @@ COVER_PROFILE=coverage.out
 # TODO(efried): Figure out how to force all go targets to
 #    gvm use go1.13.6
 
-# Include shared Makefiles
-include project.mk
-include standard.mk
-
```
Comment (2uasimojo) [inline Makefile:12]: This part was manual.

File: Makefile [hunk 2]
```diff
@@ -6,20 +9,13 @@ COVER_PROFILE=coverage.out
...
-# Build the docker image
-.PHONY: docker-build
-docker-build: build
+.PHONY: update_standards
+update_standards:
+	@standards/update
 
-# Push the docker image
-.PHONY: docker-push
-docker-push: push
```
Comment (2uasimojo) [inline Makefile:25]: manual

File: Makefile [hunk 3]
```diff
@@ -6,20 +9,13 @@ COVER_PROFILE=coverage.out
...
 .PHONY: coverhtml
 coverhtml: coverage
 	go tool cover -html $(COVER_PROFILE)
 
-# Build the docker image
-.PHONY: docker-build
-docker-build: build
+.PHONY: update_standards
+update_standards:
+	@standards/update
```
Comment (2uasimojo) [inline Makefile:20]: Manual, but part of the [documented setup](https://gitlab.cee.redhat.com/service/boilerplate/blob/31e448fcd245d983b2497d8bfbec228a53f12c10/README.md#bootstrap)

File: boilerplate/update [hunk 4]
```diff
@@ -0,0 +1,110 @@
+#!/usr/bin/env bash
```
Comment (2uasimojo) [inline boilerplate/update:1]: Initial copyin of this file was manual, per the [documented setup](https://gitlab.cee.redhat.com/service/boilerplate/blob/31e448fcd245d983b2497d8bfbec228a53f12c10/README.md#bootstrap). This should only happen once.

File: boilerplate/update.cfg [hunk 5]
```diff
@@ -0,0 +1,2 @@
+openshift/golang_osd_cluster_operator
```
Comment (2uasimojo) [inline boilerplate/update.cfg:1]: Manual. This is where conventions are opted into.

File: boilerplate/openshift/golang_osd_cluster_operator/standard.mk [hunk 6]
```diff
@@ -61,9 +61,15 @@ push:
 	docker push $(OPERATOR_IMAGE_URI)
```
Comment (2uasimojo) [inline boilerplate/openshift/golang_osd_cluster_operator/standard.mk:61]: Note that the supposed "renames" (per git) of the four files
- codecov.sh
- golint.sh
- project.mk
- standard.mk

...were actually two things:
1. The manual step of removing those files, as they're superseded by...
2. The boilerplate versions being laid down under the `boilerplate` directory hierarchy.

File: Makefile (hunk 3 above)
Comment (dustman9000) [reply Makefile:20]: "standards" -> "boilerplate"?

Comment (2uasimojo) [reply Makefile]: whoops, yeah.

**Comment by 2uasimojo** [comment]
boilerplate has moved on significantly since this PR was proposed. If/when we decide we're really going to onboard, we'll do it fresh.

**Pull request closed without merging**